Solutions/archTIS/Package/createUiDefinition.json

```diff
@@ -27,25 +27,16 @@
       }
     },
     "basics": [
-      {
-        "name": "getLAWorkspace",
```

Comment:
These codes should be present because this performs cascading between resource group and workspace. Can you please add it back.

Reply:
@NikTripathi 
I am not sure why do you want me to change this. I have followed steps from other solutions and they have same structure as I have.
When I added back getLAWorkspace code, it start failing Azure Toolkit which you shared me. So I have revert back and added previous code. Also tested Azure Toolkit validation with 

{
        "name": "workspace",
        "type": "Microsoft.Solutions.ResourceSelector",
        "label": "Log Analytics Workspace",
        "toolTip": "Select the Log Analytics Workspace to add the Azure Sentinel solution contents",
        "resourceType": "Microsoft.OperationalInsights/workspaces",
        "options": {
          "filter": {
            "subscription": "onBasics"
          }
        },
        "visible": true
      }

Please clarify why do you want me to change and if it require then how can resolve issue for Azure Toolkit validation.

Reply:
Actually because of the code you have provided in createUiDefinition we are getting the cascading as following screenshot.
![image](https://user-images.githubusercontent.com/87809732/136210951-767a5900-8b7b-4627-92e5-0c762a6df7d6.png)
But we don't want it like this. We want only those log analytics workspace which are related to specific resource group. Picture for reference.
![image](https://user-images.githubusercontent.com/87809732/136211351-4f8d2b46-4e9a-4ffc-bd00-062d59ea024c.png)

So I have made the required changes and created a zip. Please update the zip in place of zip and uiDefinition and mainTemplate also. You can compare the codes and check the changes I have made. 
Thanks.
Let me know if you need something.
[1.0.2.zip](https://github.com/Azure/Azure-Sentinel/files/7294235/1.0.2.zip)